db/schema.sql

```diff
@@ -1,8 +1,49 @@
+CREATE TABLE diaperEntries (
```

Comment:
I actually see these as being journals as well, with type diaper and value pee or poop. So only provides the ability to do one of the other for now, but makes things a little simpler.